### ISSUE TITLE
use VSlider

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/molarity#55]
Molarity was written before Slider existed.  It has its own `VerticalSlider` subclass of `Node`.   And `VerticalSlider` should be replaced by `VSlider`.  

This should be done before adding a11y or PhET-iO to this sim, so that we can leverage work done in Slider. @jessegreenberg @zepumph @samreid FYI.

**Comment by pixelzoom** [comment]
Whoopsie.  Looks like this was already done in https://github.com/phetsims/molarity/issues/40.  Nevermind :)
